common/springUtilities/teamFunctions.lua

```diff
@@ -147,6 +183,9 @@ return {
 		IsFFA          = function () return getSettings().isFFA          end,
 		---@return boolean
 		IsSandbox      = function () return getSettings().isSandbox      end,
+		---@return table? isHoliday Currently running holiday events. 
+		---See common/holidays.lua for more information.
+		IsHoliday = function () return getSettings().isHoliday end,
```

Comment:
`IsFoo` should conventionally return a bool, not a table. So like
```
Spring.Utilities.Gametype.IsHoliday("aprilfools") -> bool
```
If you want a return a table call it something like`Spring.Utilities.Gametype.GetCurrentHolidays()`.